modules/nf-core/deseq2/differential/templates/deseq_de.R

```diff
@@ -187,13 +205,13 @@ if (!opt\$contrast_variable %in% colnames(sample.sheet)) {
         'column of the sample sheet'
         )
     )
-} else if (!is.null(opt\$blocking)) {
-    blocking.vars = unlist(strsplit(opt\$blocking, split = ';'))
+} else if (!is.null(opt\$blocking_variables)) {
+    blocking.vars = unlist(strsplit(opt\$blocking_variables, split = ';'))
     if (!all(blocking.vars %in% colnames(sample.sheet))) {
         stop(
             paste0(
                 'One or more of the blocking variables specified (',
-                opt\$blocking,
+                opt\$blocking_variables,
```

Comment:
I'd suggest maybe changing this error message to mention which variables are missing? For blocking, probably people will usually not list that many variables, I understand that, but generally, I try to make such error messages as fool-proof as possible by directly pointing to the offending values

Reply:
Haha, yes, I hadn't thought the max 2 or 3 variables people could use would present a problem- but okay, I've it more explicit :-)